app/assets/stylesheets/partials/_base.scss

```diff
@@ -49,4 +49,11 @@ Common styles
    color: #6f777b;
    font-size: 27px;
    margin-bottom: 5px;
- }
\ No newline at end of file
+ }
+
+.preheader {
```

Comment:
This isn't necessary as we can use the sanctioned GDS class - maybe we can update the heading component as per this branch?
https://github.com/hmrc/binding-tariff-trader-frontend/pull/194

Reply:
Fair point! It has been removed and as per previous comment, the captionMsg has been added in the heading component! 👍